### DISCUSSION TITLE
Success with Solarman wifi dongle

### DISCUSSION THREAD

**Discussion by purcell-lab** [kellerza/sunsynk#217] [General]
A big thanks to https://www.youtube.com/watch?v=X_ypBi7iuJc&t=78s for providing a step by step guide for this configuration.

I couldn't see too much in the documentation for this config setup.

I now have monitoring and control via wifi dongle of two inverters.

![image](https://github.com/kellerza/sunsynk/assets/79175134/982d362b-66e9-402f-a024-e5092234fbd3)

![Screenshot 2023-12-29 08 36 52](https://github.com/kellerza/sunsynk/assets/79175134/9d6d1cc9-149c-4028-85c6-9ab5c2b0d41e)

**Comment by kellerza** [comment]
You're welcome to propose some additions to the docs. It seemed pretty obvious, but I do have some prior knowledge :wink:

**Comment by mdeakin99** [comment]
Interesting.  This might be the way forward for me as I am struggling currently.  Any restrictions using this method?
